internal/cli/tenants.go

```diff
@@ -166,7 +167,13 @@ func addTenantCmd(cli *cli) *cobra.Command {
 			if len(args) == 0 {
 				err := tenantDomain.Pick(cmd, &inputs.Domain, cli.tenantPickerOptions)
 				if err != nil {
-					return err
+					if !errors.Is(err, errUnauthenticated) {
+						return err
+					}
```

Comment:
In the event we get a different type of error we won't silence it accidentally. For this case, we only care about `errUnauthenticated`.